.gitignore

```diff
@@ -4,6 +4,11 @@
 *$py.class
 *.swp
 *.patch
+*.csv
```

Comment:
blocking all .csv, .txt, and vagrantfile 2 & 3 from git seems an odd choice. Are there really no cases where it would ever make sense (such as test data) to include txt files or .csv files in this project?